src/components/tl/ucp/bcast/bcast_dbt.c

```diff
@@ -114,7 +119,10 @@ void ucc_tl_ucp_bcast_dbt_progress(ucc_coll_task_t *coll_task)
             if ((t1.children[i] != UCC_RANK_INVALID) &&
                 (t1.children[i] != coll_root)) {
                 UCPCHECK_GOTO(ucc_tl_ucp_send_nb(buffer, data_size_t1, mtype,
-                                                 t1.children[i], team, task),
+                                                 ucc_ep_map_eval(
```

Comment:
Am I missing something or this loop and the next one should iterate over `args->active_set.size` elements?